modules/social_features/social_event/modules/social_event_managers/src/Plugin/views/field/SocialEventManagersViewsBulkOperationsBulkForm.php

```diff
@@ -275,8 +276,14 @@ public function viewsFormValidate(&$form, FormStateInterface $form_state) {
     if ($this->view->id() === 'event_manage_enrollments') {
       $user_input = $form_state->getUserInput();
       $available_options = $this->getBulkOptions();
+      $selected_actions = $this->options['selected_actions'];
       // Grab all the actions that are available.
-      foreach ($this->options['selected_actions'] as $action_key => $action) {
+      foreach (Element::children($this->actions) as $action) {
+        // If the option is not in our selected options, next.
+        if (($action_key = array_search($action, array_filter((array) array_combine(array_keys($selected_actions), array_column($selected_actions, 'action_id'))))) === FALSE) {
```

Comment:
For readability reasons I would recommend to split this statement in multiple variables.

Reply:
Done